build.sbt

```diff
@@ -252,7 +252,9 @@ lazy val rholang = (project in file("rholang"))
     scalacOptions ++= Seq(
       "-language:existentials",
       "-language:higherKinds",
-      "-Yno-adapted-args"
+      "-Yno-adapted-args",
+      "-Xfatal-warnings",
+      "-Xlint:_,-missing-interpolator" // disable "possible missing interpolator" warning
```

Comment:
In tests for rholang `interpolate` method we use strings of pattern `"${a}"` which makes scalac think we forgot the `s"…"` pattern.

Reply:
Well... maybe we should be using """${a}""", or sth different?

Reply:
There is one solution : `f"$${a}"`. Do you think it's better?